app/src/main/java/ar/com/wolox/android/features/homepage/news/NewsFragment.kt

```diff
@@ -10,6 +12,77 @@ class NewsFragment @Inject constructor() : WolmoFragment<NewsFragmentBinding, Ne
     override fun layout() = R.layout.news_fragment
 
     override fun init() {
+        setUpRecycler()
+    }
+
+    override fun setListeners() {
+        with(binding) {
+            newsSwipeRefresh.setOnRefreshListener {
+                newsSwipeRefresh.isRefreshing = false
```

Comment:
I think the presenter has to handle this event. Try doing something like `presenter.onSwipeRefresh()`